docs/references/subcommands/analyze/detect-vendored.md

```diff
@@ -23,6 +23,18 @@ By default, VSI ignores the following directory:
 
 ### FAQ
 
+#### Is there a limit to the number of scanned files?
+
+This feature does not have a concrete limit, however at this time FOSSA considers projects that have more than 35,000 files "unsupported".
+That being said, they may work.
+
+> [!NOTE]
+> Files are counted recursively, meaning that every archive contained in your project is unpacked and its contents are scanned,
+> and each of those interior files count when talking about the 35,000 file supported limit.
+
+If you find that this is a significant issue in your usage of this feature, we'd love to hear more about your use case-
+we're always on the lookout for how we can improve this functionality!
```

Comment:
Should we put a link to the support page here?

Reply:
good idea, done!